### ISSUE TITLE
add options 

### ISSUE THREAD

**Issue by Swift-fox82** [amir1376/ab-download-manager#305]
Hello, is there an option to automatically open the program when we want to download

**Comment by Yamazaki-wu** [comment]
Yes, just like IDM, the program will only be launched when you click to download, instead of always being in the tray after booting.

**Comment by BedBug2479** [comment]
yes this is a good idea

currently if i close abd then it doesnot auto launch but downloads in the browser
